packages/salesforcedx-vscode-core/src/commands/templates/forceLightningAppCreate.ts

```diff
@@ -64,27 +92,33 @@ const outputDirGatherer = new SelectOutputDir(AURA_DIRECTORY, true);
 const metadataTypeGatherer = new MetadataTypeGatherer(AURA_TYPE);
 
 export async function forceLightningAppCreate() {
+  const createTemplateExecutor = sfdxCoreSettings.getTemplatesLibrary()
```

Comment:
This piece of code is repeating across all templates. Can we refactor this out and have a unit test for this? We could reduce how much we test in 'Library Create' tests in all templates. Right now we are testing the templates library itself instead of the interaction.

Reply:
The 'Library Create' tests are actually adding a lot of code coverage (about +10% for each template classes). The previous tests are only covering commands construction, while the new tests are actually doing template creation and asserting that files are created, file's content etc. This is an unintended benefit but it adds 1% to the overall coverage in the vscode repo.

As for the experimental settings, I don't think there's much benefits in extracting them out at this moment, since the eventual goal is to remove the setting and it'll be a similar amount of work if it's in the form right now, or it's in the form of a small utility function.